.github/workflows/push-on-develop.yml

```
@@ -58,12 +58,21 @@ jobs:
+
+      - name: Set up Docker Buildx 🏗️
+        uses: docker/setup-buildx-action@v3
+
       - name: Build and push Docker images 🐳
         uses: docker/build-push-action@v6
         with:
           context: .
           file: ./Dockerfile
           push: true
+          platforms:
+            - linux/amd64
```

actionlint

expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)